### ISSUE TITLE
format_fasta_benchmark throws seqan::UnexpectedEnd

### ISSUE THREAD

**Issue by marehr** [seqan/seqan3#744]
When executing `./format_fasta_benchmark`, I get the following output:

```terminal
benchmark/io % ./format_fasta_benchmark 
2019-02-15 15:19:28
Running ./format_fasta_benchmark
Run on (8 X 3900 MHz CPU s)
CPU Caches:
  L1 Data 32K (x4)
  L1 Instruction 32K (x4)
  L2 Unified 256K (x4)
  L3 Unified 8192K (x1)
Load Average: 0.85, 0.59, 0.41
***WARNING*** CPU scaling is enabled, the benchmark real time measurements may be noisy and will incur extra overhead.
-----------------------------------------------------
Benchmark           Time             CPU   Iterations
-----------------------------------------------------
write3            184 ns          184 ns      3851589
write2            271 ns          270 ns      2564422
read3             841 ns          840 ns       834837
terminate called after throwing an instance of 'seqan::UnexpectedEnd'
  what():  Unexpected end of input.
[1]    21271 abort (core dumped)  ./format_fasta_benchmark
```

**Comment by h-2** [comment]
@marehr is this resolved?

**Comment by smehringer** [comment]
I remember that this was not a bug but simply a design issue of the benchmark together with marcels high-performance laptop :rocket: which read the whole file/stream within the execution time of the benchmark

**Comment by rrahn** [comment]
@marehr ping

**Comment by h-2** [comment]
fixed by #732.
